attractions/dropdown/dropdown-alignment.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This is needed to improve type inference in JSDoc, as it doesn't yet support `as const` assertions.
+ * See [microsoft/TypeScript#30445](https://github.com/microsoft/TypeScript/issues/30445) for more details.
+ * @type {{
+ *  AUTO_START: 'auto-start',
+ *  AUTO_END: 'auto-end',
+ *  START: 'start',
+ *  END: 'end',
+ * }}
+ */
+export const HorizontalAlignment = {
+  AUTO_START: 'auto-start',
+  AUTO_END: 'auto-end',
+  START: 'start',
+  END: 'end',
+};
+
+/**
+ * This is needed to improve type inference in JSDoc, as it doesn't yet support `as const` assertions.
+ * See [microsoft/TypeScript#30445](https://github.com/microsoft/TypeScript/issues/30445) for more details.
+ * @type {{
+ * AUTO_BOTTOM: 'auto-bottom',
```

Comment:
Could you please indent these lines? And the ones above, with 2 spaces from the text, currently it's just one.
```suggestion
 *   AUTO_BOTTOM: 'auto-bottom',
```